spec/event_tag_utils_spec.rb

```diff
@@ -113,8 +231,8 @@
     end
 
     it 'should return nil if event tags does not contain the numeric tag value' do
-      expect(spy_logger).to receive(:log)
-        .with(Logger::DEBUG, 'The numeric metric key is not defined in the event tags.').exactly(1).times
+      expect(spy_logger).to receive(:log).exactly(1).times
+                                         .with(Logger::DEBUG, 'The numeric metric key is not defined in the event tags.')
```

Comment:
Put this exactly(1).times after 'with' statement as in all others. 
They will object on this change. No need